File: main (1).pas
{

                            Online Pascal Compiler.
                Code, Compile, Run and Debug Pascal program online.
Write your code in this editor and press "Run" button to execute it.

}


program Hello;
var x, c, z, d, p: real;
var y: real;
begin
writeln('Enter x, c');
readln(x);
readln(c);

z:= (x*x*x + c*c) * sin(Pi / (c*x));
if z >= 0 then 
    begin
        y:= 1;
            while y <= 5 do begin
                y:= y + 0.2;
                d:= (y*y) + sqrt(y);
                
                if d > 20 then writeln(d, y)
                else writeln(x*c*d)
            end
    end
else writeln('z<0')
end.

File: main (1).cpp
/******************************************************************************

                              Online C++ Compiler.
               Code, Compile, Run and Debug C++ program online.
Write your code in this editor and press "Run" button to compile and execute it.

*******************************************************************************/

#include <iostream>

using namespace std;

int main()
{
    int year;
    cout<<"Введите год :-)";
    cin>>year;

    if(year % 100 == 0) {
        if(year % 400 == 0) {
            cout<<"Год високосный";
        }
        else cout<<"Год не високосный";
    }
    else {
        if(year % 4 == 0) {
            cout<<"Год високосный";
        }
        else cout<<"Год не високосный";
    }
}
